README.md

```diff
@@ -9,16 +9,21 @@
   * [Table of Contents](#table-of-contents)
   * [Requirements](#requirements)
   * [Usage](#usage)
+  * [Adding the Nomad Pack Registry](#adding-the-nomad-pack-registry)
+  * [Running a Nomad Pack](#running-a-nomad-pack)
+    * [Testing a Nomad Pack](#testing-a-nomad-pack)
   * [Notes](#notes)
   * [Author Information](#author-information)
   * [License](#license)
 <!-- TOC -->
 
 ## Requirements
 
-* HashiCorp Nomad `1.5.5` or [newer](https://developer.hashicorp.com/nomad/downloads).
-* HashiCorp Nomad Pack `0.0.1` or [newer](https://releases.hashicorp.com/nomad-pack/).
-* `terraform-docs` `0.16.0` or [newer](https://terraform-docs.io/user-guide/installation/).
+- HashiCorp Nomad `1.5.6` or [newer](https://developer.hashicorp.com/nomad/downloads)
```

Comment:
Is that a hard requirement? Shouldn't it be more like 1.5.0+ ?

Reply:
This may or may not have been a search-and-replace error.